app/models/manageiq/providers/ibm_power_hmc/infra_manager/template.rb

```diff
@@ -1,2 +1,31 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Template < ManageIQ::Providers::InfraManager::Template
+  supports :provisioning
+
+  def get_host(host_id)
+    ManageIQ::Providers::IbmPowerHmc::InfraManager::Host.find(host_id)
+  end
```

Comment:
This doesn't make much sense as an instance method (especially a public one) since it isn't using any instance-level attributes.  If you have an exact host_id you could just do `Host.find(options[:host_id])` in the method below